data/test_e2ebot/config.yml

```diff
@@ -0,0 +1,16 @@
+language: en
+pipeline:
+  - name: WhitespaceTokenizer
+    intent_tokenization_flag: True
+  - name: RegexFeaturizer
+  - name: LexicalSyntacticFeaturizer
+  - name: CountVectorsFeaturizer
+  - name: CountVectorsFeaturizer
+    analyzer: char_wb
+    min_ngram: 1
+    max_ngram: 4
+  - name: DIETClassifier
+    epochs: 5
+policies:
+- name: TEDPolicy
+  epochs: 5
```

Comment:
we can't use less epochs for either of them, no?

Reply:
Thought this was the lowest, but now seems like I can get down to `1` for DIET and `4` for `TED`.